DotNetWebApi/Controllers/LocationController.cs

```diff
@@ -0,0 +1,87 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using DotNetWebApi.Models;
+
+namespace DotNetWebApi.Controllers;
+
+[Route("[controller]")]
+[ApiController]
+public class LocationController : ControllerBase
+{
+    private readonly TeddyBearsContext _context;
+
+    public LocationController(TeddyBearsContext context)
+    {
+        _context = context;
+    }
+
+    [HttpGet("")]
```

Comment:
Nit: This won't change functionality but you can remove `("")` from the http attributes for cleaner code and to keep people from thinking a value was missed.